spotifyutils/__init__.py

```diff
@@ -51,6 +52,31 @@ def cli(input_args=None):
     args = parser.parse_args(input_args)
     main(**vars(args))
 
-
 def main(**kwargs):
-    print('Test')
+    """ Direct user to authorize spotify URL """
+
+    configfile = kwargs['configfile']
+    
+    config = configparser.ConfigParser()
+    config ['spotifyutils'] = {
+        'ClIENT_ID': kwargs['client_id'],
+        'CLIENT_SECRET': kwargs['client_secret'],
+        'REDIRECT_URI': kwargs['redirect_uri']
+    }
+
+    with open(configfile, "w") as f:
+        config.write(f)
+
+    parser = configparser.ConfigParser()
+    parser.read(configfile)
+
+    client_id = parser.get('spotifyutils', 'client_id')
+    client_secret = parser.get('spotifyutils', 'client_secret')
+    redirect_uri = parser.get('spotifyutils', 'redirect_uri')
```

Comment:
Consider the configuration precedence. The configuration file is one of the places that configuration can be set. By convention, the precedence should be

1. command line arguments
2. config file
3. argument defaults.

Reply:
See the above comment, CLI takes precedence, although I got rid of the defaults. Should I add those back in?